summingbird-scalding/src/main/scala/com/twitter/summingbird/scalding/ScaldingPlatform.scala

```diff
@@ -564,6 +566,44 @@ object Scalding {
   }
 
   /**
+   * like toPipeExact, but using Execution
+   */
+  def toExecutionExact[T](
+    dr: DateRange,
+    prod: Producer[Scalding, T],
+    options: Map[String, Options] = Map.empty): Execution[TypedPipe[(Timestamp, T)]] =
+
+    Execution.getMode.flatMap { mode =>
+      val fd = new FlowDef
+      toPipeExact(dr, prod, options)(fd, mode) match {
+        case Right(_) =>
+          /**
+           * We plan twice. The first time we plan to see what new
+           * writes we do, the second time we only read, we do not
+           * do any new writes.
+           */
+          Execution.fromFn { (_, _) => fd }
+            .flatMap { _ =>
+              // Now plan again and use summingbird's built in support
+              // to minimize work by reading existing on-disk data:
+              val reads = new FlowDef
+              toPipeExact(dr, prod, options)(reads, mode) match {
```

Comment:
Forgive me, I'm very unfamiliar with the SB codebase. Don't we have the expectation that `Config` options provided to an `Execution` could affect the behavior of how the Scalding Job runs? Or is that supposed to be represented in `options`?